src/core/Akka.Remote/Endpoint.cs

```diff
@@ -1385,7 +1385,7 @@ public sealed class FlushAndStop
             public static FlushAndStop Instance { get { return _instance; } }
         }
 
-        public sealed class AckIdleCheckTimer
+        public sealed class AckIdleCheckTimer: IAutoReceivedMessage
```

Comment:
Im not a fan of this. Its an internal message, and its seems very random to only mark this message as `IAutoReceivedMessage`. (Which would cause it to also have `IDeadLetterSuppression`).